### DISCUSSION TITLE
Box64 on ARM Acer Chromebook R13

### DISCUSSION THREAD

**Discussion by JezzDaBezz** [ptitSeb/box64#94] [Ideas]
Could I use Box 64 on Acer Chromebook r13? 
![image](https://user-images.githubusercontent.com/84666756/129627860-fd5bffb5-e7f0-46e1-a85d-98fc93681f76.png)

**Comment by USMP-lancer** [comment]
i got it working in the chromeos internal vm, you just have to install needed dependencies
